clientapi/jsonerror/jsonerror.go

```diff
@@ -124,6 +124,12 @@ func GuestAccessForbidden(msg string) *MatrixError {
 	return &MatrixError{"M_GUEST_ACCESS_FORBIDDEN", msg}
 }
 
+// UnsupportedRoomVersion is an error which is returned when the client
+// requests a room with a version that is unsupported.
+func UnsupportedRoomVersion(msg string) *MatrixError {
```

Comment:
Please link to the spec for this error code.

Reply:
I appreciate it the other errors don't - they should be. The purpose is to make clear that these are Matrix spec error codes and not made up Dendrite error codes.

Reply:
There is a link at the top by `type MatrixError struct` which defines these error codes :-)

Reply:
Very well! Ignore me then :)